lib/network.js

```diff
@@ -31,4 +51,4 @@ const receiveBody = async (req) => {
   return Buffer.concat(buffers);
 };
 
-module.exports = { fetch, receiveBody };
+module.exports = { fetch, receiveBody, makeRequestOptions };
```

Comment:
We do not need to export it separately
```suggestion
module.exports = { fetch, receiveBody };
```

Reply:
That's what I thought too, but then its not possible to test it